models/issues/review_list.go

```diff
@@ -107,8 +107,12 @@ func (opts *FindReviewOptions) toCond() builder.Cond {
 	if opts.ReviewerID > 0 {
 		cond = cond.And(builder.Eq{"reviewer_id": opts.ReviewerID})
 	}
-	if opts.Type != ReviewTypeUnknown {
-		cond = cond.And(builder.Eq{"type": opts.Type})
+	if len(opts.Types) > 0 {
+		typeConds := make([]builder.Cond, 0, len(opts.Types))
+		for _, reviewType := range opts.Types {
+			typeConds = append(typeConds, builder.Eq{"type": reviewType})
```

Comment:
use `builder.In`?

Reply:
a34862aadf1d2cb1b71a4daeb0c71bb50853f7db